event_types/event_types.py

```diff
@@ -779,17 +803,18 @@ def split_data_train_test(dtf_e, test_size=0.75, random_state=777):
 
     Returns
     -------
-    Two dictionaries of DataFrames, one for training and one for testing
+    Two nested dictionaries of DataFrames, one for training and one for testing
     (keys=energy ranges, values=separated DataFrames).
     """
 
     dtf_e_train = dict()
     dtf_e_test = dict()
 
-    for this_e_range, this_dtf in dtf_e.items():
-        dtf_e_train[this_e_range], dtf_e_test[this_e_range] = model_selection.train_test_split(
-            this_dtf, test_size=test_size, random_state=random_state
-        )
+    for this_e_range in dtf_e.keys():
+        for this_offset_range, this_dtf in dtf_e[this_e_range].items():
```

Comment:
Are we careful here not to divide the training dtf in offset bins? I guess we showed we don't want to train in offset bins.

Reply:
Yes, we're not binning in offset the training sample, it's just one big bin including everithing. That line is just to enter that offset bin and look what is inside. In fact, that function saves the train and test samples with only energy bins, not even the big offset bin, because it's not necessary for the training nor the testing (adding the prediction column).

Reply:
OK, could you maybe add a comment in the code above it just so that it's clear? (I will definitely forget about it in 2 weeks if I look at it again).